examples/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[workspace]
+members = [
```

Comment:
Could this just be `members = ["*"]`?

Reply:
If we do this, I think we'll need to add an `exclude = ["target"]` as well.

Reply:
Perhaps we could rearrange the subcrates of `http-rust-outbound-http` to be flat with the rest of the examples?

Reply:
Sounds fine to me -- let's do that in a separate PR, though.